package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "github-actions-allow-list-as-code-action",
   "version": "1.2.0",
```

Comment:
We should bump this to 2.0.0

Reply:
Done - https://github.com/ActionsDesk/github-actions-allow-list-as-code-action/pull/79/commits/a52f2183e896760770aa6e71552d78a13feba44d